app/components/account/AccountContainer.js

```diff
@@ -0,0 +1,107 @@
+// import redux, react
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import { Link, browserHistory } from 'react-router';
+
+//Material UI
+import { StepsTextField, StepsRaisedButton, StepsFlatButton } from '../material-style'
+import { Paper } from 'material-ui';
+import { background, primary } from '../colors';
+import moment from 'moment';
+
+// -=-=-=-=-=-= COMPONENT =-=-=-=-=-=-
+
+export class AccountInfo extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {}
+  }
+
+  render() {
+
+    const {user} = this.props;
+
+    return (
+
+      <div id="account-info" className="col-xs-12" > 
+        <Helmet title="Account Info" />
```

Comment:
I'm guessing you set this up to be modified into a form at some point? Confused me that all the info was in editable fields, but this isn't actually configured to edit/update. Fine by me. This is certainly not a must have for our demo, just wanted to confirm your thinking.